.github/workflows/test.yml

```diff
@@ -0,0 +1,31 @@
+name: Test
+on: [push, pull_request]
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node: [8, 10, 12]
```

Comment:
Note to self, we'll bump this later as part of https://github.com/Level/community/issues/98.